src/components/BookList.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-named-as-default */
```

Comment:
- [ ] Kindly ensure you do not disable linters as they help us write cleaner code.